test/integration-test/openmldb-test-java/openmldb-deploy/src/main/resources/deploy.properties

```diff
@@ -0,0 +1,25 @@
+
+zk_url=https://archive.apache.org/dist/zookeeper/zookeeper-3.4.14/zookeeper-3.4.14.tar.gz
+
+main=https://github.com/4paradigm/OpenMLDB/releases/download/v0.2.3/openmldb-0.2.3-linux.tar.gz
+0.2.2=https://github.com/4paradigm/OpenMLDB/releases/download/0.2.2/openmldb-0.2.2-linux.tar.gz
+0.2.3=https://github.com/4paradigm/OpenMLDB/releases/download/v0.2.3/openmldb-0.2.3-linux.tar.gz
+spark_url=https://github.com/4paradigm/spark/releases/download/v3.0.0-openmldb0.3.2/spark-3.0.0-bin-openmldbspark.tgz
+
+tmp=http://pkg.4paradigm.com:81/rtidb/test/openmldb-0.5.3-linux.tar.gz
+tmp_zk_url=http://pkg.4paradigm.com:81/rtidb/test/zookeeper-3.4.14.tar.gz
+tmp_spark_url=http://pkg.4paradigm.com:81/rtidb/test/spark-3.0.0-bin-openmldbspark-v052.tgz
+
+single=http://pkg.4paradigm.com:81/rtidb/test/openmldb-0.5.3-linux.tar.gz
+single_zk_url=http://pkg.4paradigm.com:81/rtidb/test/zookeeper-3.4.14.tar.gz
```

Comment:
no inner url

Reply:
Test environment deployment, cicd will not be used